resset/action.go

```diff
@@ -44,11 +44,20 @@ const (
 	// not modifying other attributes. In practice, this mostly applies to
 	// starting/stopping/signaling machines.
 	ActionControl
+
+	// ActionSign, ActionVerify, ActionEncrypt, and ActionDecrypt indicate
+	// operations using keys.
+	ActionSign
+	ActionVerify
+	ActionEncrypt
+	ActionDecrypt
```

Comment:
@btoews are you OK having this sort of KMS-specific detail in the macaroon library?
Would it be better to have some generic names here like "ActionOp1", "ActionOp2" and then in petsem authority define those names as "ActionSign", "ActionVerify"?

Reply:
Another thing that works well with having a second, extended `Action` type is that it's more backwards compatible with existing macaroons. For `Encrypt`, for example, we can require `ActionControl | ActionEncrypt`. When checking the `Apps` caveat, we'll only look for the `ActionControl` part, but we'll look for `ActionControl` and `ActionEncrypt` when verifying `Secrets` or `SecretTypes` caveats.